src/test/java/org/kainos/ea/validator/LoginValidatorTest.java

```diff
@@ -3,43 +3,43 @@
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.kainos.ea.cli.Login;
-import org.kainos.ea.core.LoginValidator;
+import org.kainos.ea.core.Validator;
 
 import static org.junit.jupiter.api.Assertions.*;
 
 public class LoginValidatorTest {
```

Comment:
this file can be removed as its duplicate of src/test/java/org/kainos/ea/service/ValidatorServiceTest.java